components.yaml

```diff
@@ -37,10 +37,10 @@ components:
     metadata: ""
   multus-dynamic-networks:
     url: https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller
-    commit: 467e7a2528450e300bc27c884add0ae9a21ee135
+    commit: fcc5f6b304176217802112e8aac051d5452644e6
     branch: main
     update-policy: static
```

Comment:
can we change update-policy to tagged on stable branches U/S for multus dynamic ?
it will also allow auto cherry-picking

/cc @maiqueb